neurons/validator/validator.py

```diff
@@ -361,14 +298,6 @@ async def run(self):
         self.step = 0
         while True:
             try:
-                # Reduce calls to miner to be approximately 1 per 5 minutes
```

Comment:
Do we have a sleep or async sleep here so we don't iterate the loop too quickly if there aren't any organic requests to pull?

Reply:
@tavurth

Reply:
@Supreme-Emperor-Wang yes we do, the section:

```py
                # NOTE: Will wait for around 30 seconds
                #       trying to get a task from the user
                #       before going on and creating a synthetic task
                task = await get_task(self.api_url)
```

I've added a note for clarify